content/posts/Arnab_Das_GSoC_phase1.md

```diff
@@ -0,0 +1,178 @@
+---
+title: 'CircuitVerse VueJS Simulator | GSoC@2023 | Phase-1 Report'
+date: 2023-07-10T00:13:11+05:30
+draft: false
+author: Arnab Das
+type: post
+---
+
+![GSoC@CircuitVerse](/images/Arnab_GSoC23/cover.png)
+
+This blog post is to summarize the work done in the First Phase of **Google Summer of Code @ 2023**.
+
+### [Work Repository 🖥](https://github.com/CircuitVerse/cv-frontend-vue)
+
+> **[_CircuitVerse Simulator with VueJS -_](https://summerofcode.withgoogle.com/programs/2023/projects/eg17qD6w)
+> While carrying forward the goals from last year's project My project focuses on finishing up the development of Vue.js circuit simulator while boosting its performance, stability, and scalability by transitioning the JavaScript codebase to TypeScript. This project encompasses integrating the simulator with the main repository, carrying out new API endpoint creation and integration, introducing new features, and enhancing user experience.**
```

Comment:
Just a small suggetion, these blogs are from our side to the community, kindly use we, us instead of `I`. 😄